### DISCUSSION TITLE
ADD Clippy Checks and cargo audit to this repo and rustdesk-server

### DISCUSSION THREAD

**Discussion by dinger1986** [rustdesk/rustdesk#6226] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### SubTasks

https://github.com/actions-rs/clippy-check

This can be setup as a github action to check any PRs

https://github.com/marketplace/actions/rust-audit-check

This will check for any vulnerabilities in dependencys alerting to any issues

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk-server/pull/82

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk-server/pull/271#issue-1785609507

**Comment by dinger1986** [comment]
I don't know if I can do this, I can get the steps in place on one of my repos but it might not pr across

**Comment by dinger1986** [comment]
have added both here https://github.com/rustdesk/rustdesk/pull/4868

**Comment by dinger1986** [comment]
I closed the PR cause it kept failing but only failing due to it running checks on files it shouldnt so needs decided how it should run I guess and then I can reconfigure and do a new PR
